templates/asset_form.html.twig

```diff
@@ -42,6 +51,24 @@
                   ) }}
                {% endif %}
 
+               {% if item.isField('num') %}
+                  {{ fields.autoNameField(
+                     "num",
+                     item,
+                     _x('phone', 'Number'),
+                     withtemplate
+                  ) }}
+               {% endif %}
+
+               {% if item.isField('power_supplies') %}
```

Comment:
if it's possible to change, a `field.numberField` has a better ux (it may require to add support of max/min/step keys)